src/internal/modules/invoice-accounts/forms/select-address.js

```diff
@@ -3,24 +3,28 @@
 const Joi = require('@hapi/joi');
 const urlJoin = require('url-join');
 const { formFactory, fields } = require('shared/lib/forms/');
+const titleCase = require('title-case');
 
 const address = (address) => {
   return [
-    address.addressLine1,
-    address.addressLine2,
-    address.addressLine3,
-    address.addressLine4,
-    address.town,
+    titleCase(address.addressLine1),
+    titleCase(address.addressLine2),
+    titleCase(address.addressLine3),
+    titleCase(address.addressLine4),
+    titleCase(address.town),
     address.postcode
   ].filter(item => item !== '' && item !== null);
```

Comment:
I think that you can use the lodash method `isEmpty` here:
`.filter(item => !isEmpty(item));

Reply:
done